<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lotus</class>
 <widget class="QWidget" name="Lotus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="btn_min">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>5</y>
     <width>26</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="btn_close">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>5</y>
     <width>26</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="avatar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>96</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 170, 255);
border-color: rgb(0, 170, 255);
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>单击此处
设置头像</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>username</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="search">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>133</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="TabLabel" name="friendLabel" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>163</y>
     <width>96</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="TabLabel" name="roomLabel" native="true">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>163</y>
     <width>96</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="TabLabel" name="otherLabel" native="true">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>163</y>
     <width>96</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="QTreeView" name="friendTree">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>200</y>
     <width>290</width>
     <height>451</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>653</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QListView" name="roomList">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>290</width>
     <height>450</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QTreeView" name="otherView">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>200</y>
     <width>290</width>
     <height>450</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QWidget" name="bg" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>290</width>
     <height>682</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/lotus_bg.png);</string>
   </property>
   <widget class="QLabel" name="aboutButton">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>657</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pixmap/image/aboutButton.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="friendSearch">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="servername">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>servername</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="StatusTextLine" name="statusText">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>bg</zorder>
  <zorder>btn_min</zorder>
  <zorder>btn_close</zorder>
  <zorder>avatar</zorder>
  <zorder>username</zorder>
  <zorder>search</zorder>
  <zorder>friendLabel</zorder>
  <zorder>roomLabel</zorder>
  <zorder>otherLabel</zorder>
  <zorder>friendTree</zorder>
  <zorder>searchButton</zorder>
  <zorder>roomList</zorder>
  <zorder>otherView</zorder>
  <zorder>friendSearch</zorder>
  <zorder>servername</zorder>
  <zorder>statusText</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabLabel</class>
   <extends>QWidget</extends>
   <header>tablabel.h</header>
  </customwidget>
  <customwidget>
   <class>StatusTextLine</class>
   <extends>QLineEdit</extends>
   <header>statustextline.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
